demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -0,0 +1,221 @@
+            const float *det_conf_data = reinterpret_cast<float*>(in_ssd_conf.data);
+            const float *prior_data    = reinterpret_cast<float*>(in_ssd_priorbox.data);
+
+            std::vector<float*> action_conf_data;
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor1.data));
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor2.data));
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor3.data));
+            action_conf_data.push_back(reinterpret_cast<float*>(in_ssd_anchor4.data));
+                
+            const cv::Size frame_size = in_frame.size();
+            std::unique_ptr<ActionDetector> detector(new ActionDetector);
+            detector->GetPostProcResult(local_data, det_conf_data, prior_data, action_conf_data, frame_size, out_detections);
+        }
+    };
+}// namespace custom
+    
+int main(int argc, char* argv[]) {
+    cv::CommandLineParser cmd(argc, argv, keys);
+    if (cmd.has("help")) {
+        cmd.printMessage();
+        return 0;
+    }
+
+    cv::GComputation pp([]() {
+        cv::GMat in;
+        cv::GMat detections = cv::gapi::infer<custom::Faces>(in);
+        cv::GArray<cv::Rect> faces = custom::PostProc::on(detections, in);
+
+        cv::GMat location;
+        cv::GMat detect_confidences;
+        cv::GMat priorboxes;
+        cv::GMat action_con1;
+        cv::GMat action_con2;
+        cv::GMat action_con3;
+        cv::GMat action_con4;
+
+        std::tie(location, detect_confidences, priorboxes, 
+                 action_con1, action_con2, action_con3, action_con4) = cv::gapi::infer<custom::PersAction>(in);
+
+        cv::GArray<Detections> persons = custom::PersonDetActionRec::on(location, detect_confidences, priorboxes,
+                                                                action_con1, action_con2, action_con3, action_con4,
+                                                                in);
+
+        cv::GMat frame = cv::gapi::copy(in);
+        return cv::GComputation(cv::GIn(in), cv::GOut(frame, faces, persons));
+    });
+
+
+    std::string fdPathXML = cmd.get<std::string>("fdm"); 
+    auto det_net = cv::gapi::ie::Params<custom::Faces> {
+        fdPathXML,
+        getBinPath(fdPathXML),
+        cmd.get<std::string>("fdd"),
+    };
+
+    std::string paPathXML = cmd.get<std::string>("pam"); 
+    auto pos_act_net = cv::gapi::ie::Params<custom::PersAction> {
+        paPathXML,
+        getBinPath(paPathXML),
+        cmd.get<std::string>("fdd"),
+    }.cfgOutputLayers({"mbox_loc1/out/conv/flat", 
+                       "mbox_main_conf/out/conv/flat/softmax/flat",
+                       "mbox/priorbox",
+                       "out/anchor1",
+                       "out/anchor2",
+                       "out/anchor3",
+                       "out/anchor4"
+                       }); // 0006 have another outputs
```

Comment:
person-detection-action-recognition-0006 has another outputs (small changes in post processing).